openjdk9/src/main/docker/Dockerfile

```diff
@@ -35,10 +35,6 @@ RUN \
     netbase \
     unzip \
 
- # Remove libgnutls30 dependency brought in by openjdk package. One of its dependencies uses
- # the AGPL license, so we cannot use it.
- && dpkg --remove --force-depends libgnutls30 \
```

Comment:
While, we're here, we should probably also add `wget` above to be consistent with `openjdk:8`.